napari/layers/__init__.py

```diff
@@ -5,11 +5,14 @@
 to the super constructor.
 """
 
-
+from ..utils.misc import all_subclasses
 from .base import Layer
 from .image import Image
+from .labels import Labels
 from .points import Points
-from .vectors import Vectors
-from .surface import Surface
 from .shapes import Shapes
-from .labels import Labels
+from .surface import Surface
+from .vectors import Vectors
+
+names = [subclass.__name__.lower() for subclass in all_subclasses(Layer)]
```

Comment:
I think this should be all caps (i.e., a constant)?
```suggestion
NAMES = [subclass.__name__.lower() for subclass in all_subclasses(Layer)]
```